src/libraries/System.Private.Xml/src/System/Xml/Schema/XmlSchemaComplexContentRestriction.cs

```diff
@@ -2,16 +2,17 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+#nullable enable
 namespace System.Xml.Schema
 {
     using System.Collections;
     using System.Xml.Serialization;
 
     public class XmlSchemaComplexContentRestriction : XmlSchemaContent
     {
-        private XmlSchemaParticle _particle;
-        private XmlSchemaObjectCollection _attributes = new XmlSchemaObjectCollection();
-        private XmlSchemaAnyAttribute _anyAttribute;
+        private XmlSchemaParticle? _particle;
+        private XmlSchemaObjectCollection? _attributes = new XmlSchemaObjectCollection();
```

Comment:
Can be non-nullable.

Reply:
This is a public API which have corresponding property which allow to set it to null, I think this is more correct representation of state - assuming all callsites expect it to be non-null should we annotate this based on what makes more sense or to represent the state? I.e. we've annotated ToString as nullable even though 99.9% of the code assumes it's not null

Reply:
```csharp
        internal void SetAttributes(XmlSchemaObjectCollection newAttributes)
        {
            _attributes = newAttributes;
        }
```

doesn't do any validation - will check if this can be non-nullable

Reply:
How about adding an assert and triggering a CI run to see if anything breaks?